riscv/entropy_source.h

```diff
@@ -53,8 +53,8 @@ class entropy_source {
     if(return_status == OPST_ES16) {
 
         // Add some sampled entropy into the low 16 bits
-        uint16_t seed  = this -> get_two_random_bytes();
-        result        |= seed;
+        uint16_t entropy  = this -> get_two_random_bytes();
+        result        |= entropy;
```

Comment:
The formatting here is weird; just use this:

```
        uint16_t entropy = this -> get_two_random_bytes();
        result |= entropy;
```

(As a separate matter, this entire file uses a different formatting discipline from the rest of Spike, but fixing that should be done in another PR.)